server/etc/e-smith/templates/etc/postfix/smtpd_sender_login_maps/10UsersFromSSSD

```diff
@@ -0,0 +1,14 @@
+{
+  use utf8;
+  use JSON;
+  binmode STDOUT, ":utf8";
+  my $json;
+
+  $json = `/usr/libexec/nethserver/list-users`;
```

Comment:
A remote network call is not allowed from a template. We must rely on the information provided by e-smith DB contents